src/lib/actions/movies.ts

```diff
@@ -0,0 +1,63 @@
+"use server";
+
+import { prisma } from "@/lib/prisma";
+import { revalidatePath } from "next/cache";
+import { MovieSchema } from "@/lib/definitions";
+
+export interface FormState {
+  message: string;
+}
+
+export async function addMovie(prevState: FormState, formData: FormData) {
```

Comment:
myśle że lepiej jakby user wysyłał tylko id z the movie database i czy chce zobaczyć albo widział a reszte danych pobierać z api tmdb a nie brać z formData